regen/feature.pl

```diff
@@ -244,9 +245,9 @@ sub longest {
 $::bundle, $::feature
 .
 
-for ('default', sort grep /\.\d[02468]/, keys %feature_bundle) {
-    $::bundle = ":$_";
-    $::feature = join ' ', @{$feature_bundle{$_}};
+foreach my $bundle ('default', sort grep /\.\d*[02468]\z/a, keys %feature_bundle) {
```

Comment:
The `\z` in here looks wrong. It will skip the "5.9.5" three-component one.

Reply:
you are right about the `\z` but removing it will have no effects
note that `5.9.5` bundle is currently not listed in `feature.pm`
view https://metacpan.org/pod/feature#FEATURE-BUNDLES 
are you saying that this is a bug in blead and we should fix it?
Do you still think `\z` should be removed? I'm good with both options